src/app/frontend/replicasetlist/logsmenu_controller.js

```diff
@@ -80,4 +87,15 @@ export default class LogsMenuController {
         },
         (err) => { this.log_.error('Error fetching Replica Set pods: ', err); });
   }
+
+  /**
+   * @param {string} podId
+   * @param {string} container
```

Comment:
The var names should probably be podName and containerName, because this is what they really are, correct?